From 0f1711cec719010e643a9cfc58863edec5c24102 Mon Sep 17 00:00:00 2001
From: Alessandro Rubini <rubini@gnudd.com>
Date: Wed, 22 Oct 2014 17:08:57 +0200
Subject: [PATCH 03/11] irq: export one symbol for external irq controller

    - Cherry-picked from v2.6.39 patch
    - Fixed the conflicts

The conflicts happened because the symbols we need are already exported.

irq_to_desc has been exported by Jiri Kosina in v3.5 and
set_irq_flags has been exported by Arnd Bergmann in v3.7.

Signed-off-by: Alessandro Rubini <rubini@gnudd.com>
---
 kernel/irq/handle.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/kernel/irq/handle.c b/kernel/irq/handle.c
index 131ca17..330d954 100644
--- a/kernel/irq/handle.c
+++ b/kernel/irq/handle.c
@@ -33,6 +33,7 @@ void handle_bad_irq(unsigned int irq, struct irq_desc *desc)
 	kstat_incr_irqs_this_cpu(irq, desc);
 	ack_bad_irq(irq);
 }
+EXPORT_SYMBOL_GPL(handle_bad_irq);
 
 /*
  * Special, empty irq handler:
-- 
1.7.7.2

